docs/guide.md

```diff
@@ -236,7 +236,7 @@ Here is a complete list of the project variables defined by this template:
   - Example
 - - `project_name`
   - Project name on PyPI and GitHub repo name
-  - `ssb-library`
+  - null
```

Comment:
@mallport I think the example column should still have the `ssb-library`-entry. It is an example of how to fill in the requested value, and `ssb-library` is such an example. It also gives a hint to the SSB GitHub naming standard describing that SSB libraries shall start with the `ssb-` prefix. The default value is changed to null, and that's fine, but I think we should still keep the example.